versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/installing-kubeslice/registering-the-worker-cluster.mdx

```diff
@@ -7,9 +7,9 @@ register your worker clusters with the KubeSlice Controller.
 ## Prerequisites
 Before you begin, ensure the following prerequisites are met:
 
-- You have the KubeSlice Controller installed on a separate cluster. For more information, see [Installing the KubeSlice Controller](/versioned_docs/version/getting-started-with-cloud-clusters/installing-kubeslice/installing-the-kubeslice-controller.mdx).
-- You have set up the environment for the worker clusters. For more information, see [Prerequisites](/versioned_docs/version/getting-started-with-cloud-clusters/prerequisites/prerequisites.mdx).
-- You have authenticated the worker clusters with the cloud providers. For more information, see [Preparing the clusters for registration](/versioned_docs/version/getting-started-with-cloud-clusters/prerequisites/preparing-the-cloud-clusters-for-registeration.mdx).
+- You have the KubeSlice Controller installed on a separate cluster. For more information, see [Installing the KubeSlice Controller](/versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/installing-kubeslice/installing-the-kubeslice-controller.mdx).
+- You have set up the environment for the worker clusters. For more information, see [Prerequisites](/versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/prerequisites/prerequisites.mdx).
+- You have authenticated the worker clusters with the cloud providers. For more information, see [Preparing the clusters for registration](/versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/prerequisites/preparing-the-cloud-clusters-for-registeration.mdx).
```

Comment:
change `registeration.mdx` to `registration.mdx` in the filename

Reply:
This has been updated already. The changes have been pushed to the branch DOC-broken-links.